polaris-core/src/main/java/org/apache/polaris/core/storage/StorageAccessProperty.java

```diff
@@ -39,6 +41,18 @@ public enum StorageAccessProperty {
       Boolean.class, "s3.path-style-access", "whether to use S3 path style access", false),
   CLIENT_REGION(
       String.class, "client.region", "region to configure client for making requests to AWS"),
+  AWS_REFRESH_CREDENTIALS_ENABLED(
+      Boolean.class,
+      AwsClientProperties.REFRESH_CREDENTIALS_ENABLED,
+      "whether to enable automatic refresh of credentials",
+      true,
+      false),
+  AWS_REFRESH_CREDENTIALS_ENDPOINT(
+      String.class,
+      AwsClientProperties.REFRESH_CREDENTIALS_ENDPOINT,
+      "the endpoint to use for refreshing credentials",
```

Comment:
ideally we are not refreshing the credentials, we are getting brand new one how about we say something like 
https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml#L119